CONTRIBUTING_DOCS.md

```diff
@@ -158,24 +151,51 @@ Here are some other shortcodes:
 - `readfile`: Include the content of another file in the current file, which can be in an arbitrary location.
 - `bootstrap-table`: formats a table using Bootstrap classes; accepts any bootstrap table classes as additional arguments, e.g. `{{< bootstrap-table "table-bordered table-hover" }}`
 
+### Add code to documentation pages
+
+For command, binary and process names, you can use a pair of backticks (\`\<some-name\>\`): `<some-name>`.
+
+Larger blocks of multi-line code text such as configuration files can be wrapped in triple backticks, with a language as a parameter for highlighted formatting.
+
+You can also use the `ghcode` shortcode to embed a single file directly from GitHub:
```

Comment:
I did not know this one!

Reply:
The original issue didn't mention documenting this: I was the driver to get the shortcode in the first place, and I think I'm the only person who's been using it since it wasn't documented anywhere...